libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/hwdef.dat

```diff
@@ -0,0 +1,238 @@
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_AEROFOX_H7
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 128
+
+# UART1 is debug
+PA9 USART1_TX USART1 NODMA
+PA10 USART1_RX USART1 NODMA
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_ESCTelemetry
+
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1# crystal frequency
+
+# supports upto 8MBits/s
+CANFD_SUPPORTED 8
+
+# with 2M flash we can afford to optimize for speed
+env OPTIMIZE -O2
+
+# system timer
+STM32_ST_USE_TIMER 5
+
+# UART
+SERIAL_ORDER OTG1 UART7 UART4 UART5 USART2 USART1 UART8 USART3 OTG2
+
+#serial1
+PA15 UART7_TX UART7
+PA8 UART7_RX UART7
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_ESCTelemetry
+define DEFAULT_SERIAL1_BAUD 115200
+
+# serial2
+PC10 UART4_TX UART4
+PC11 UART4_RX UART4
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_None
+define DEFAULT_SERIAL2_BAUD 115200
+
+# serial3
+PC12 UART5_TX UART5
+PD2 UART5_RX UART5
+define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_None
+define DEFAULT_SERIAL3_BAUD 115200
+
+# serial4
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_None
+define DEFAULT_SERIAL4_BAUD 115200
+
+# serial6
+PE1 UART8_TX UART8
+PE0 UART8_RX UART8
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_MAVLink2
+define DEFAULT_SERIAL6_BAUD 115200
+
+# serial7
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_RCIN
+define DEFAULT_SERIAL7_BAUD 115200
+
+# default the 2nd interface to SLCAN
+define HAL_OTG2_PROTOCOL SerialProtocol_SLCAN
```

Comment:
we prefer that the second OTG use MAVLINK2 (the default) to avoid user confusion...using MAVLink SLCAN s better access to CAN